mix.exs

```diff
@@ -21,7 +21,7 @@ defmodule PhoenixRest.Mixfile do
   end
 
   defp deps do
-    [{:phoenix, "~> 1.2"},
+    [{:phoenix, "~> 1.3.0-rc"},
      {:plug_rest, "~> 0.11"},
```

Comment:
Hi @christopheradams, sorry had a frantic week.
I tested it on my pet project, should be good!

Thanks for merging~ = )